src/pmt.cpp

```diff
@@ -362,7 +370,7 @@ int del_filter(int id) {
     f->enabled = 0;
     mutex_unlock(&filters_mutex);
     mutex_destroy(&f->mutex);
-    LOG("deleted filter %d, ad %d, pid %d, max filters %d", id, adapter, pid,
+    LOG("deleted filter %d, ad %d, pid %d, num filters %d", id, adapter, pid,
```

Comment:
I'll drop this change before merging, the semantics are not correct (`nfilters` represents the next unassigned filter, not the number of filters)